core/project_settings.cpp

```diff
@@ -738,10 +734,7 @@ Error ProjectSettings::_save_settings_text(const String &p_file, const Map<Strin
 	Error err;
 	FileAccess *file = FileAccess::open(p_file, FileAccess::WRITE, &err);
 
-	if (err) {
-		ERR_EXPLAIN("Couldn't save project.godot - " + p_file);
-		ERR_FAIL_COND_V(err, err);
-	}
+	ERR_FAIL_COND_V_MSG(err, err, "Couldn't save project.godot - " + p_file + ".");
```

Comment:
Not very important, but I'd make the first `err` check more explicit: `err != OK`.